mifospay/src/main/java/org/mifos/mobilewallet/mifospay/base/BaseActivity.java

```diff
@@ -88,7 +88,7 @@ public void showProgressDialog(String message) {
     @Override
     public void hideProgressDialog() {
         if (progressDialog != null) {
-            progressDialog.hide();
+            progressDialog.dismiss();
```

Comment:
why u added this  in the `public void hideProgressDialog()` at `BaseActivity.java.` if u want to add progressBar.dismiss() then just create `public void hideProgressDialog()` method.
@devansh-299  please check it..